Source/Initialization/DivCleaner/ProjectionDivCleaner.cpp

```diff
@@ -326,7 +326,7 @@ WarpX::ProjectionCleanDivB() {
     } else if ( WarpX::electromagnetic_solver_id == ElectromagneticSolverAlgo::Yee
             ||  WarpX::electromagnetic_solver_id == ElectromagneticSolverAlgo::HybridPIC
             ||  ( (WarpX::electrostatic_solver_id == ElectrostaticSolverAlgo::LabFrame
-                || WarpX::electrostatic_solver_id == ElectrostaticSolverAlgo::LabFrameElectroMagnetostatic)
+                || WarpX::electrostatic_solver_id == ElectrostaticSolverAlgo::LabFrameSemiImplicit)
```

Comment:
No, the magnetostatic solver is just changed to function as an add-on to an electrostatic solver. So instead of defining it as it's own solver (explicit ES + MS) the type of ES solver is set in the input and the extra flag `do_magnetostatic` sets whether the MS solve will be added on to the ES solve.